helper/dataGeneration-Mongo.js

```diff
@@ -0,0 +1,51 @@
+const fs = require('fs');
+const writeItems = fs.createWriteStream('mongoImages.csv');
+
+let globalCounter = 99
+let numberOfEntries = 10000100
+let pictureId = 1
+
+// Items 1-2000 in loremflickr are cat photos, so looping through them with a simple counter
+// will create 2000 unique animal images
+
+console.time('Time to Generate Data: ');
+
+writeItems.write('itemId,pic1Small,pic1Med,pic1Large,pic2Small,pic2Med,pic2Large,\n', 'utf8');
```

Comment:
What if there is an error writing? 

writeItems is a writable stream, I know the node docs are @.@ but it didn't take much to find https://nodejs.org/docs/latest-v12.x/api/stream.html#stream_event_error

Anything that is asynchronous needs to have error handling!